src/Process.php

```diff
@@ -101,6 +97,14 @@ public function __construct($cmd, $cwd = null, array $env = null, array $fds = n
             );
         }
 
+        if (DIRECTORY_SEPARATOR === '\\') {
+            foreach ($fds as $fd) {
+                if (isset($fd[0]) && $fd[0] === 'pipe') {
+                    throw new \LogicException('Windows isn\'t supported due to the blocking nature of STDIN/STDOUT/STDERR pipes.');
```

Comment:
^ that

Reply:
Good catch! I've updated the exception message and updated the documentation to mention a `LogicException`.

It's my understanding that `LogicException` is a better fit here because `BadMethodCallException` refers to the method invocation. The invocation itself is now valid on Windows (so a `BadMethodCallException` would have been reasonable in the previous version), now it's just that we can't fulfill the request due to blocking pipes on Windows.